subsys/logging/log_rpc_internal.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I guess everything what isn't in public API folder is internal. How much sense is in `internal` suffix?

Reply:
I added also a public `log_rpc.h` header so I wanted to make it faster to find the correct file and easier to distinguish open tabs in VS Code. I also found a few other examples of that in `nrf/subsys/` so I didn't think I would break any convention.

Reply:
They are distinguished by C including rules.
#include <nrf_rpc.h> is always public since it is passed with system paths.
#include "nrf_rpc.h" is always internal, since developer explicitly said to compiler to look for it in the current folder.

but up to you if it is easier to develop with word suffix, let it be.

Reply:
I'm not saying there's any issues for the compiler. Rather for me when searching for the right file :). But I will rename it to `log_rpc_group.h` as this is what it actually provides.